modules/oe_translation_poetry/src/Form/AddLanguagesRequestForm.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_poetry\Form;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\Url;
+
+/**
+ * Form for requesting new translations.
```

Comment:
This is not very representative. How about something like "Form for requesting new languages for an existing translation request"